Default argument is not working for max-iterations

#178 seems to have introduced a bug on this repo, we should fix

```
Run cd /tmp && DEBUG=true python -m openhands_resolver.resolve_issue \
DEBUG:openhands:DEBUG mode enabled.
19:17:40 - openhands:DEBUG: logger.py:[23](https://github.com/All-Hands-AI/openhands-resolver/actions/runs/11671094113/job/32496719650#step:8:24)6 - Logging initialized
19:17:40 - openhands:DEBUG: logger.py:[24](https://github.com/All-Hands-AI/openhands-resolver/actions/runs/11671094113/job/32496719650#step:8:25)8 - Logging to file in: /opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/logs
usage: resolve_issue.py [-h] --repo REPO [--token TOKEN] [--username USERNAME]
                        [--runtime-container-image RUNTIME_CONTAINER_IMAGE]
                        [--max-iterations MAX_ITERATIONS] --issue-number
                        ISSUE_NUMBER [--output-dir OUTPUT_DIR]
                        [--llm-model LLM_MODEL] [--llm-api-key LLM_API_KEY]
                        [--llm-base-url LLM_BASE_URL]
                        [--prompt-file PROMPT_FILE]
                        [--repo-instruction-file REPO_INSTRUCTION_FILE]
                        [--issue-type {issue,pr}]
resolve_issue.py: error: argument --max-iterations: expected one argument
Error: Process completed with exit code 2.
```

--- a/.github/workflows/openhands-resolver-experimental.yml
+++ b/.github/workflows/openhands-resolver-experimental.yml
@@ -70,7 +70,7 @@ jobs:
             echo "ISSUE_NUMBER=${{ github.event.issue.number || inputs.issue_number }}" >> $GITHUB_ENV
             echo "ISSUE_TYPE=issue" >> $GITHUB_ENV
           fi
-          echo "MAX_ITERATIONS=${{ inputs.max_iterations }}" >> $GITHUB_ENV
+          echo "MAX_ITERATIONS=${{ inputs.max_iterations || 50 }}" >> $GITHUB_ENV
 
       - name: Comment on issue with start message
         uses: actions/github-script@v7

--- a/.github/workflows/openhands-resolver.yml
+++ b/.github/workflows/openhands-resolver.yml
@@ -70,7 +70,7 @@ jobs:
             echo "ISSUE_NUMBER=${{ github.event.issue.number || inputs.issue_number }}" >> $GITHUB_ENV
             echo "ISSUE_TYPE=issue" >> $GITHUB_ENV
           fi
-          echo "MAX_ITERATIONS=${{ inputs.max_iterations }}" >> $GITHUB_ENV
+          echo "MAX_ITERATIONS=${{ inputs.max_iterations || 50 }}" >> $GITHUB_ENV
 
       - name: Comment on issue with start message
         uses: actions/github-script@v7

